me.gladwell.eclipse.m2e.android/src/main/java/me/gladwell/eclipse/m2e/android/AndroidMavenProjectConfigurator.java

```diff
@@ -71,7 +71,7 @@ public void configure(ProjectConfigurationRequest request, IProgressMonitor moni
                     .getProject());
 
             if (mavenProject.isAndroidProject()) {
-                if (shouldCallJavaProjectConfigurator()) {
+                if (!usesM2E1_6OrNewer()) {
```

Comment:
Well, we test against the `true` case here: https://github.com/rgladwell/m2e-android/pull/363/files#diff-b7c4af84c9011346c7842073feca3d61R26